core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -873,4 +873,17 @@ class PlaintextConsumerTest extends BaseConsumerTest {
       waitTimeMs=leaveGroupTimeoutMs
     )
   }
+
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedGroupProtocolNames)
+  @MethodSource(Array("getTestGroupProtocolParametersAll"))
+  def test(groupProtocol: String): Unit = {
```

Comment:
Could you change a meaningful naming? Thanks.

Reply:
Yeah, that's right, I have changed the test name, PTAL